### ISSUE TITLE
Excessive dynamic airway collapse 

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7012]
**Preferred term label:**
Excessive dynamic airway collapse

**Synonyms**
Membranous tracheobronchomalacia,  EDAC

**Definition (free text, please give PubMed ID)**
Excessive anterior displacement of the tracheal and/ or bronchial membranous wall

**Parent term (use hpo.jax.org/app)**
Tracheobronchomalacia HP:0002786

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
